File: csharp/OrderProcessingService/model/Order.cs
using System.ComponentModel.DataAnnotations;

namespace OrderProcessingService.model
{
    public class Order {
        public string orderName {get; set;}
        public int orderNum {get; set;}

        public Order(string orderName, int orderNum) {
            this.orderName = orderName;
            this.orderNum = orderNum;
        }
    }
}

File: service_invocation/java/OrderProcessingService/src/main/java/com/service/OrderProcessingService/model/Order.java
package com.service.OrderProcessingService.model;

public class Order {
    String orderName;
    Integer orderNum;

    public Order(String orderName, Integer orderNum) {
        this.orderName = orderName;
        this.orderNum = orderNum;
    }

    public String getOrderName() {
        return orderName;
    }

    public void setOrderName(String orderName) {
        this.orderName = orderName;
    }

    public Integer getOrderNum() {
        return orderNum;
    }

    public void setOrderNum(Integer orderNum) {
        this.orderNum = orderNum;
    }

    @Override
    public String toString() {
        return "Order{" +
                "orderName='" + orderName + '\'' +
                ", orderNum='" + orderNum + '\'' +
                '}';
    }
}
